package.json

```diff
@@ -17,7 +17,8 @@
     "react-scripts": "3.4.1",
     "redux": "^4.0.5",
     "redux-thunk": "^2.3.0",
-    "typescript": "~3.7.2"
+    "typescript": "~3.7.2",
+    "yarn": "^1.22.4"
```

Comment:
Let's delete this line: `"yarn": "^1.22.4"`

Reply:
done